packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { FileProperties } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
+import {
+  ExtensionContext,
+  Memento
+} from 'vscode';
+
+interface ConflictFileProperties {
+  lastModifiedDate: string;
+}
+
+export class PersistentStorageService {
+  private storage: Memento;
+  private static instance?: PersistentStorageService;
+
+  private constructor(context: ExtensionContext) {
+    this.storage = context.workspaceState;
+  }
+
+  public static initialize(context: ExtensionContext) {
+    PersistentStorageService.instance = new PersistentStorageService(context);
+  }
+
+  public static getInstance(): PersistentStorageService {
+    if (!PersistentStorageService.instance) {
+      throw new Error('Storage should have been initialized upon extension activation');
```

Comment:
Let's localize this message, we can do that by adding the message to [this file](https://github.com/forcedotcom/salesforcedx-vscode/blob/844a6497b46400b5556a0c4eb97d52b863b978bb/packages/salesforcedx-vscode-core/src/messages/i18n.ts)

Reply:
Thanks for pointing me to that file. I made an addition, let me know if that works